server/command.go

```diff
@@ -65,6 +67,10 @@ func getAutocompleteData() *model.AutocompleteData {
 	show := model.NewAutocompleteData("show", "", "Show MS Teams linked channel")
 	cmd.AddCommand(show)
 
+	sync := model.NewAutocompleteData("sync", "[number-of-hours]", "Sync the current channel with MS Teams channel (by default 24 hours)")
+	sync.RoleID = model.SystemAdminRoleId
```

Comment:
Does this conflict with the `model.PermissionManageSharedChannels` permission check below?

Reply:
good point, the idea here was to not expose the slash commands if there were not executable, but in this case, makes a lot of sense of reqmove this RoleID restriction.